subsys/bluetooth/controller/ll_sw/lll.h

```diff
@@ -208,6 +214,13 @@ struct node_rx_pdu {
 enum {
 	EVENT_DONE_EXTRA_TYPE_NONE,
 	EVENT_DONE_EXTRA_TYPE_CONN,
+/* Following proprietary defines must be at end of enum range */
+#if defined(CONFIG_BT_PROPRIETARY_EXT)
+	EVENT_DONE_EXTRA_TYPE_PROPRIETARY_START,
+	EVENT_DONE_EXTRA_TYPE_PROPRIETARY_END =
+		EVENT_DONE_EXTRA_TYPE_PROPRIETARY_START + 0x5,
```

Comment:
0x5 should be a kconfig.

Reply:
I added a config (BT_CTLR_USER_EXT _RANGE) in subsys/bluetooth/controller/Kconfig